cicd/push-release-tag-ghub.sh

```diff
@@ -0,0 +1,50 @@
+################ This script is customized for use on PNNL's internal CI platform
```

Comment:
Many of the CICD files reference our internal CI - @bonicim, just confirming that this all does actually want to be merged into the public repo?

Reply:
Yeah, this was something that I had issues about. We want our Stash and Github repos to mirror each other but since we are using an internal CI tool (i.e. Bamboo) we're exposing to the public the inner workings of our CI process. I've ensured that any sensitive information are referenced by [environment variables such as this example](https://github.com/pnnl/HyperNetX/blob/develop-3c910af/cicd/push-release-tag-ghub.sh#L31). I've seen other projects include their CI scripts that reference using CI tools such as TravisCI so I don't think this is too big a deal. @brendapraggastis What are your thoughts?